data/packages/fishtown-analytics/adwords/index.json

```diff
@@ -5,5 +5,6 @@
     "latest": "0.2.12",
     "assets": {
         "logo": "logos/placeholder.svg"
-    }
+    },
+    "redirectnamespace": "dbt-labs"
```

Comment:
i think these `index.json` files will be overwritten every time the [hubcap script](https://github.com/dbt-labs/hubcap/blob/master/hubcap.py#L258-L261) runs. This makes a ton of sense as a backfill, but should we also update the hubcap script to either:
1. Avoid overwriting this index file entirely (we'd still need to update the `latest` key)
2. Set the `redirectnamespace` and `redirectname` for these packages

Reply:
Good point @drewbanin! I'm not sure that hubcap will be overwriting these old folders I've been updating -- that's kind of the problem 😅  Hubcap only updates the latest location of the package, and not these older folders which are the ones that have redirects. 

That being said, I'm sure we'll need to spend more time thinking about better naming conventions and where this redirect information should eventually live to be picked up by Core's dbt deps. I'm 💯 on doing iterations on this format after we apply the short term fix.

Reply:
🤦 🤦 🤦 yes..... you're so right..... thanks for helping me get there :)